### PR TITLE
quickstart fixes

### PR DESCRIPTION
**Pull request by phaer**
trying to fix formatting issues and clarify it a bit, incorporating feedback by @chaoflow at oceansprint

### CODE REVIEW

**Comment by chaoflow** [comment]
@phaer https://github.com/nix-community/nixos-anywhere/blob/main/docs/quickstart.md#steps-required-to-run-nixos-anywhere --- Step 2, ``nix flake init`` seems to not be needed, I suggest to instead explain to copy all files from the example repo, or use flake template as discussed today.

cc @Mic92

**Comment by phaer** [comment]
@chaoflow Ah right, I was a bit lazy yesterday and sleep-deprived enough that I wasn't sure whether my changes would be a net-positive in terms of readability. Still not totally recovered, but that might be good enough?

**Comment by phaer** [comment]
Didn't really get what you meant here: an scp command? L

**Comment by phaer** [comment]
I thought that was almo

File: docs/quickstart.md [hunk 1]
```diff
@@ -70,11 +63,12 @@ below.
   ```
 
   This creates the necessary configuration files under `/mnt/etc/nixos/`, which
-  you can then customize as needed.
+  you can then customize as needed and copy them to your local machine in order
```
Comment (Mic92) [inline docs/quickstart.md:66]: Ah, you kind of mentioning this here. I missed that sentence on my phone.

**Comment by Mic92** [comment]
@mergify queue

File: docs/quickstart.md (hunk 1 above)
Comment (phaer) [reply docs/quickstart.md:66]: Ah, just saw that I somehow posted a butchered response from my phone, hope that wasn't confusing

**Pull request merged**